src/Symfony/Component/PropertyInfo/Util/PhpDocTypeHelper.php

```diff
@@ -39,7 +40,7 @@ public function getTypes(DocType $varType): array
                 $nullable = true;
             }
 
-            $type = $this->createType((string) $varType, $nullable);
+            $type = $this->createType($varType, $nullable);
             if (null !== $type) {
```

Comment:
you can inline this IMO

Reply:
Yeah since you are not modifying it, forgot about this ;). It will be harder to merge things back to master.

Reply:
Yes, I tried to modify as few lines as possible, to keep diffs shorts.